src/ioctls/vm.rs

```diff
@@ -2305,6 +2462,47 @@ mod tests {
         vm_fd.set_irq_line(0x02_00_0010, true).unwrap();
     }
 
+    #[test]
+    #[cfg(target_arch = "riscv64")]
+    #[allow(clippy::unusual_byte_groupings)]
```

Comment:
nit: I think this was copied from another test - I don't see any unusual byte grouping in this test.

Reply:
Ah yes, I reference the test on aarch64, I'll remove this :)